### PR TITLE
ui: temp splash and loading screen

### PR DESCRIPTION
**Pull request by nicole-obrien**
## Summary

> Please summarize your changes, describing **what** they are and **why** they were made.

...

### Platforms

> Please select any platforms where your changes have been tested.

-   **Desktop**
    -   [ ] MacOS
    -   [ ] Linux
    -   [ ] Windows

### Instructions

> Please describe the specific instructions, configurations, and/or test cases necessary to **test and verify** that your changes work as intended.

...

## Checklist

> Please tick the following boxes that are relevant to your changes.

-   [ ] I have followed the contribution guidelines for this project
-   [ ] I have performed a self-review of my own code
-   [ ] I have commented my code, particularly in hard-to-understand areas
-   [ ] I have added or modified tests that prove my changes work as intended
-   [ ] I have verified that new and existing unit tests pass locally with my changes
-   [ ] I have verified that my latest changes pass CI workflows for testing and linting
-   [ ] I have made corresponding changes to the documentation

### CODE REVIEW

File: packages/desktop/views/Splash.svelte [hunk 1]
```diff
@@ -1,23 +1,38 @@
...
-<div class="w-full h-full flex justify-center items-center bg-white dark:bg-gray-900">
-    <div class="w-1/3">
-        <Animation classes="w-full h-auto" animation="splashscreen-desktop" loop={false} renderer="canvas" />
+<LoggedOutLayout>
+    <div slot="content" class="animate-pulse pr-100">
+        <Logo width="400" logo={LogoName.BloomLogoFull} />
     </div>
-</div>
+</LoggedOutLayout>
+```
```
Comment (MarkNerdi) [inline packages/desktop/views/Splash.svelte]: remove

File: packages/shared/src/components/LoadingScreen.svelte [hunk 2]
```diff
@@ -1,13 +1,23 @@
...
     export let progress = 0
 </script>
 
-<div class={'flex flex-col justify-center align-center items-center w-full h-full'}>
-    <Animation classes="h-64 w-64" animation="loading-desktop" loop={true} renderer="canvas" />
-    <div class="flex-col justify-center items-center w-full max-w-lg">
-        <Progress size="lg" showLabel {progress} />
+<div class={'flex flex-col justify-center w-full h-full'}>
+    <!-- <Animation classes="h-64 w-64" animation="loading-desktop" loop={true} renderer="canvas" /> -->
+    <div class="column flex flex-col items-center w-full">
```
Comment (MarkNerdi) [inline packages/shared/src/components/LoadingScreen.svelte:11]:
```suggestion
    <div class="column flex flex-col items-center w-full gap-12">
```

**Review by MarkNerdi — CHANGES_REQUESTED** [review]
can we now also remove the `loading-desktop.json`, `splashscreen-desktop.json` and `splashscreen-desktop-darkmode.json`?

File: packages/shared/src/components/LoadingScreen.svelte (hunk 2 above)
Comment (nicole-obrien) [reply packages/shared/src/components/LoadingScreen.svelte:11]: I don't know why but gap wasn't working here only in stylesheet 😢

**Review by MarkNerdi — APPROVED** [review]

**Pull request merged**